server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -240,7 +242,7 @@ public void run() {
             return mStats != null ? mStats : new MergeStats(new MergeInfo());
           });
           TabletGoalState goal = manager.getGoalState(tls, mergeStats.getMergeInfo());
-          TServerInstance location = tls.getServer();
+          Location location = tls.getLocation();
```

Comment:
Later in the code the location is logged and the Location class does not seem to have a toString method, so with this change will need to add a toString method to the Location class or change the log stmt.